source/GlassView.Benchmark/GlassView.Benchmark.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
```

Comment:
we should probably put this in a build.prop

Reply:
Done, using props ;-)